ocaml/idl/datamodel_vtpm.ml

```diff
@@ -0,0 +1,62 @@
+(*
+   Copyright (C) Citrix Systems Inc.
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU Lesser General Public License as published
+   by the Free Software Foundation; version 2.1 only. with the special
+   exception on linking described in file LICENSE.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Lesser General Public License for more details.
+ *)
+
+open Datamodel_types
+open Datamodel_common
+open Datamodel_roles
+
+let get_contents =
+  call ~name:"get_contents" ~in_product_since:"rel_next"
```

Comment:
The result is a string but the name is plural, suggesting some collection. I would have use `content`; an alternative would be `secret` - because this is what this data is.